nuxt.config.js

```diff
@@ -14,13 +14,16 @@ module.exports = NuxtHelpers([
       { hid: 'description', name: 'description', content: 'Nuxt.js project' }
     ],
     link: [
-      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }
+      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' },
+      { rel: 'stylesheet', href: 'https://fonts.googleapis.com/css?family=Montserrat:200,400' },
```

Comment:
I prefer [importing these via Sass and setting variables](https://github.com/cul-it/mann-wagon/blob/7f7453daea022c196a1421a43fe4f6a15e6b29f9/src/scss/_config.scss#L1-L6), but okay with this approach for now.